src/models/gaze17/repairs.md

```diff
@@ -0,0 +1,205 @@
+
+This model supports up to two M.2 SSDs. Both M.2 slots are size 2280. The inner-most slot supports PCIe NVMe Generation 4, and the outer-most slot supports PCIe NVMe Generation 3 or SATA.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the M.2 drive:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Unscrew the retainer screw opposite the M.2 slot.
+
+![M.2 slots](./img/m2-slots.webp)
+
+3. Remove the existing M.2 drive by pulling it out of the slot.
+4. Insert the new M.2 drive into the slot and hold it in place.
+5. Replace the retainer screw.
+
+## Replacing the wireless card:
+
+Your Gazelle's WiFi and Bluetooth are both handled by the same module. It is a standard M.2 2230 slot with PCIe and USB interfaces (A Key).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the WiFi/Bluetooth module:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Locate the wireless module. Remove any clear tape that is securing the wires.
+
+![Wireless card](./img/wireless-card.webp)
+
+3. Gently remove the two antennas (highlighted red above) by pulling them up and away from the wireless card.
+4. Remove the retaining screw opposite the M.2 slot, highlighted yellow above.
+5. The wireless card will pop up at an angle. Remove the card from the M.2 slot.
+6. Insert the new wireless card into the M.2 slot at an angle.
+7. Replace the retaining screw.
+8. Attach the two antennas by aligning the circular fittings and pressing onto the wireless card. The connectors will snap into place. _Use caution when attaching the connectors; the pins can bend, break, or snap._
+
+## Replacing the battery:
+
+The battery provides primary power whenever the system is unplugged.
+
+**Tools required:** None  
+**Time estimate:** 1 minute  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to replace the battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove the five battery screws, highlighted yellow below.
+
+![Battery](./img/battery.webp)
+
+3. Unplug the battery connector (highlighted red above) and lift the battery out of the chassis.
+
+## Replacing the CMOS battery:
+
+The CMOS battery supplies power to the system's CMOS chip. UEFI settings and the computer's hardware clock are stored on the CMOS. If your system doesn't boot, you can reset the CMOS to force a low-level hardware reset. If your clock is constantly resetting, it's likely your CMOS battery needs to be replaced.
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 15 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the CMOS battery:
```

Comment:
Is there a documented plug type if Cx needs to replace the 2-wire CMOS battery? Online there seems to be ~3 types of connectors that won't all fit into the MB

Reply:
No there is not.